gtk4/Cargo.toml

```diff
@@ -35,6 +35,7 @@ unsafe-assume-initialized = []
 # Versions from https://gitlab.gnome.org/GNOME/gnome-build-meta/-/tree/gnome-43/elements/sdk
 # gtk takes care of setting the versions of gsk and gdk
 # gio takes care of setting the version of glib
+gnome_48 = ["v4_18", "gio/v2_84", "gnome_47"]
```

Comment:
needs to be added to https://github.com/gtk-rs/gtk4-rs/tree/main/gtk4#features-1 as well

Reply:
You're right, sorry :grimacing: